test/acceptance/features/bindAppToService.feature

```diff
@@ -269,9 +270,171 @@ Feature: Bind an application to a service
             status:
                 ready: true
             """
-        Then Secret "binding-request-backend" contains "CustomReady" key with value "true"
+        Then Secret should be updated
+        Then Secret contains "CustomReady" key with value "true"
 
 
+    Scenario: Backend Service status update gets propagated to the binding secret for Volume Mount
```

Comment:
Scenario description does not reflect what is really tested: `Backend Service status update gets propagated to the binding secret for Volume Mount`. Updates are propagated to the application, not to binding secret - (binding secret is just an implementation details not interesting to users, especially now when for each update we create a new secret). Also mentioning volume mount is implementation detail - users is interested in injecting binding into application context as files.